test/upgrade/create-in-latest_version-csv-with-header.td

```diff
@@ -0,0 +1,20 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ file-append path=upgrade-csv-with-headers.csv
+id,value
+1,person
+
+> CREATE SOURCE csv_upgrade
+  FROM FILE '${testdrive.temp-dir}/upgrade-csv-with-headers.csv'
+  FORMAT CSV WITH HEADER
+
+> CREATE SOURCE csv_upgrade_explicit
+  FROM FILE '${testdrive.temp-dir}/upgrade-csv-with-headers.csv'
+  FORMAT CSV WITH HEADER (id, value)
```

Comment:
Yes, you are allowed to share a csv file between the "before" and "after" portion of the upgrade test. Testdrive is run such that the two .td tests will share the same temp-dir.

So if you can add a ```check-from``` .td file that checks the data that was ingested that would be much appreciated. If ```SHOW SOURCE``` does not return any variable strings, you can also add such a statement to the ```check-from`` .td file.

Reply:
right, that makes sense, done I think.